src/tasks/AppleAppBuilder/AppleAppBuilder.cs

```diff
@@ -223,6 +223,7 @@ public void ValidateRuntimeSelection()
 
     public override bool Execute()
     {
+        bool shouldStaticLink = true;
```

Comment:
Is this related ?

Reply:
Yeah, some version of xcode changed simulator behavior where if any dylib is deployed with the app it'll try to load it first.  This was causing problems because we always static link the components.